crates/worker/src/partition_processor_manager.rs

```diff
@@ -251,6 +254,65 @@ impl PartitionProcessorHandle {
     }
 }
 
+/// RPC message handler for Partition Processor management operations.
+pub struct PartitionProcessorManagerMessageHandler {
+    processors_manager_handle: ProcessorsManagerHandle,
+}
+
+impl PartitionProcessorManagerMessageHandler {
+    fn new(
+        processors_manager_handle: ProcessorsManagerHandle,
+    ) -> PartitionProcessorManagerMessageHandler {
+        Self {
+            processors_manager_handle,
+        }
+    }
+}
+
+impl MessageHandler for PartitionProcessorManagerMessageHandler {
+    type MessageType = CreateSnapshotRequest;
+
+    async fn on_message(&self, msg: Incoming<Self::MessageType>) {
+        info!("Received '{:?}' from {}", msg.body(), msg.peer());
```

Comment:
Intentional? if so, does this need to be at INFO level?

Reply:
It was, but happy to turn it down or eliminate altogether. Seemed like something that happens infrequently enough that it wouldn't hurt to log - but perhaps debug is much more appropriate here.